src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1010,7 +1010,7 @@ private function registerPropertyInfoConfiguration(array $config, ContainerBuild
 
         $loader->load('property_info.xml');
 
-        if (class_exists('phpDocumentor\Reflection\ClassReflector')) {
+        if (class_exists('phpDocumentor\Reflection\Types\ContextFactory')) {
```

Comment:
We should also check for existence of the `phpDocumentor\Reflection\DocBlockFactoryInterface`.